src/main/java/com/cardinity/rest/URLUtils.java

```diff
@@ -39,6 +39,11 @@ public static String buildUrl(Resource resource, UUID resourceId) {
         return buildCardinityUrl(resource, resourceId, null, null);
     }
 
+    public static String buildUrl(Resource resource, Resource action) {
+        return Cardinity.API_BASE + URL_SEPARATOR + Cardinity.API_VERSION +
```

Comment:
I think we could reuse `buildCardinityUrl(Resource resource, UUID resourceId, Resource action, UUID actionId)` method instead of building the URL here, maybe some refactoring will be needed